Typescript error : loading.close() : error TS2554: Expected 1 arguments, but got 0.

<!-- PLEASE READ THE FOLLOWING INSTRUCTIONS -->

<!--
- Try to search for your issue, it may have already been answered or even fixed in the development version.
- Check if the issue is reproducible with the latest stable version of Oruga.
- It is **required** that you clearly describe the steps necessary to reproduce the issue.
- It is recommended that you make a screenshots/animated GIFs/JSFiddle/JSBin/Codepen to demonstrate your issue.
- If your issue is resolved but still open, don’t hesitate to close it. In case you found a solution by yourself, it could be helpful to explain how you fixed it.
- Use English for communication
-->

### Overview of the problem

**Oruga** version: 0.8.5
**Vuejs** version: 3.4.19
**OS/Browser**: -

### Description

When I upgrade to oruga 0.8, I have type error.

### Steps to reproduce

src/pages/MyPage.vue:150:22 - error TS2554: Expected 1 arguments, but got 0.

150    loadingComponent.close()
                         ~~~~~~~
  node_modules/@oruga-ui/oruga-next/dist/types/types/programmatic.d.ts:18:11
    18     close(args: object): void;
                 ~~~~~~~~~~~~

### Expected behavior

As in the documentation, you must be able to call the function close() without any parameters

https://github.com/oruga-ui/oruga/blob/9693d7c63246eb0d6fab791f272b6d9574dab253/packages/docs/.vitepress/theme/examples/loading/programmatically.vue#L16

```diff
function openLoading() {
    const loadingComponent = oruga.loading.open({
        fullPage: isFullPage.value,
        container: isFullPage.value ? null : elementRef.value,
    });
+    setTimeout(() => loadingComponent.close(), 3 * 1000);
}
```

### Actual behavior

Typescript error: error TS2554: Expected 1 arguments, but got 0. 

node_modules/@oruga-ui/oruga-next/dist/types/types/programmatic.d.ts:18:11
    18     close(args: object): void;
                 ~~~~~~~~~~~~

---

Many thanks for the new version 0.8.x, the developer experience with typescript is much better.

--- a/packages/oruga/src/types/programmatic.ts
+++ b/packages/oruga/src/types/programmatic.ts
@@ -18,6 +18,6 @@ export interface ProgrammaticInstance<T = ComponentInternalInstance> {
  * because the exposed data isn't included.
  */
 export interface ProgrammaticExpose {
-    close(args: object): void;
-    promise: Promise<void>;
+    close(...args: any[]): void;
+    promise: Promise<unknown>;
 }

